docs/src/pages/premium-themes/paperbase/Header.tsx

```diff
@@ -132,10 +132,10 @@ function Header(props: HeaderProps) {
         elevation={0}
       >
         <Tabs value={0} textColor="inherit">
-          <Tab textColor="inherit" label="Users" />
-          <Tab textColor="inherit" label="Sign-in method" />
-          <Tab textColor="inherit" label="Templates" />
-          <Tab textColor="inherit" label="Usage" />
```

Comment:
These were redundant. `Tabs` is already passing these via cloneElement: https://github.com/mui-org/material-ui/blob/bc34f9aa5a5a42f47e9b97d0ab0bf49dbfa94ad0/packages/material-ui/src/Tabs/Tabs.js#L412-L420